.eslintrc

```diff
@@ -0,0 +1,32 @@
+{
+	"extends": [
+		"eslint:recommended",
```

Comment:
Can we use airbnb standards ("airbnb-base")?
What do you think @nataliepina ?

Reply:
I think that's too opiniated. For most cases it will get job done. We will have to turn rules off/on from airbnb. Though we can look into that option as well.